src/MySqlConnector/Serialization/ErrorPayload.cs

```diff
@@ -21,7 +21,7 @@ public static ErrorPayload Create(PayloadData payload)
 			reader.ReadByte(Signature);
 
 			var errorCode = reader.ReadUInt16();
-			reader.ReadByte(0x23);
+			reader.ReadByte(0x23, 0x48);
```

Comment:
I think the real problem here is that I hard-coded `0x23`. The [documentation](https://dev.mysql.com/doc/internals/en/packet-ERR_Packet.html) just defines it as "# marker of the SQL State". I'm not sure where I got the value `0x23`; maybe I just observed that in practice and assumed it was a constant?

It does not appear to be a constant :smile: and the documentation doesn't specify the set of allowed values, so I would revert the changes to `ReadByte` and just change this line to `var sqlStateMarker = reader.ReadByte();` (and discard the value).